.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  call-test-workflow:
+    uses: BlueBrain/SSCxEModelExamples/.github/workflows/python-app.yml@main
+
+  build-and-publish:
+    name: Build and make a GitHub release
+    runs-on: ubuntu-latest
+    needs: call-test-workflow
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+
+      - name: Fetch the tag, if any. If not tagged, create a patch tag
+        uses: anothrNick/github-tag-action@1.64.0
+        if: ${{ !startsWith(github.ref, 'refs/tags/') }}
+        id: tag
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          WITH_V: false
+          DEFAULT_BUMP: patch
+
+      - name: Get and store tag from 'Bump version and push tag' step
```

Comment:
Not very important, but you don't have any 'Bump version and push tag' step, which can be misleading.

Reply:
Ohhh thanks let me update that text